### ISSUE TITLE
fix: Non-focusable element in ProfileMenu

### ISSUE THREAD

**Issue by jloeffelholz** [microsoft/fluentui-blazor#4091]
# 🐛 Bug Report

The ProfileMenu component is not accessible to keyboard users. It does not have any focusable elements by default and thus cannot be accessed by using the tab key.

## 💻 Repro or Code Sample
Can be reproduced in any of the ProfileMenu examples -> https://www.fluentui-blazor.net/ProfileMenu

## 🤔 Expected Behavior

The component should be focused when a user traverses a page with tabs.

## 😯 Current Behavior

The component is skipped when using keyboard navigation via the tab key.

## 💁 Possible Solution

Use some kind of focusable element or register an onkeylistener to the menu element and all elements within the menu that users would expect to navigate to.

## 🔦 Context

Found while doing accessibility testing for my site where I'm using this component. For now, I will not be able to use this component and will need to implement a custom version of it that is accessible.

**Comment by vnbaaij** [comment]
> Found while doing accessibility testing for my site where I'm using this component. For now, I will not be able to use this component and will need to implement a custom version of it that is accessible.

Instead of creating a custom version, why not create a PR to fix the existing component?

**Comment by vnbaaij** [comment]
Addressed with PR mentioned
